### PR TITLE
Update, compress, and credit images used in the Archeo theme

### LINKED ISSUE #5433: Archeo: Pre-launch: screenshot.png
@kjellr / @beafialho - do you have a screenshot for Archeo we can use?

### LINKED ISSUE #5477: Archeo: Update all images in patterns
Once all the Archeo patterns have been merged (https://github.com/Automattic/themes/issues/5429), we should see if we can compress/optimize any of the included images.

### PR DESCRIPTION
**Pull request by kjellr**
- Updated all the images. 
- Consolidated a couple images.
- Added image credits.
- Updated theme screenshot.

`chahk.png` is still pretty large, but I was unable to find a way to scale it down to a reasonable size while maintaining a decent appearance. 😕

### CODE REVIEW

File: archeo/inc/patterns/layout-with-two-images-and-text.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<!-- /wp:paragraph -->
 	
 	<!-- wp:image {"align":"left","sizeSlug":"full","linkDestination":"none"} -->
-	<div class="wp-block-image"><figure class="alignleft size-full"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/desire-charnay.jpg" alt="' . __( 'Photograph by Désiré Charnay, 1862 to 1863', 'archeo' ) . '"/><figcaption><mark style="background-color:rgba(0, 0, 0, 0)" class="has-inline-color has-background-color"><em>' . __( 'Photograph by Désiré Charnay, 1862 – 1863', 'archeo' ) . '</em></mark></figcaption></figure></div>
+	<div class="wp-block-image"><figure class="alignleft size-full"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/pyramid.jpg.jpg" alt="' . __( 'Photograph by Désiré Charnay, 1862 to 1863', 'archeo' ) . '"/><figcaption><mark style="background-color:rgba(0, 0, 0, 0)" class="has-inline-color has-background-color"><em>' . __( 'Photograph by Désiré Charnay, 1862 – 1863', 'archeo' ) . '</em></mark></figcaption></figure></div>
```
Comment (jffng) [inline archeo/inc/patterns/layout-with-two-images-and-text.php]:
```suggestion
	<div class="wp-block-image"><figure class="alignleft size-full"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/pyramid.jpg" alt="' . __( 'Photograph by Désiré Charnay, 1862 to 1863', 'archeo' ) . '"/><figcaption><mark style="background-color:rgba(0, 0, 0, 0)" class="has-inline-color has-background-color"><em>' . __( 'Photograph by Désiré Charnay, 1862 – 1863', 'archeo' ) . '</em></mark></figcaption></figure></div>
```

**Review by jffng — COMMENTED** [review]
One typo, other than that LGTM.

**Comment by jffng** [comment]
Weird I can't link https://github.com/Automattic/themes/issues/5477 in the sidebar, but this should close that issue as well.

**Review by mikachan — APPROVED** [review]
This all looks great to me, thanks @kjellr!

**Comment by kjellr** [comment]
Thank you both!

**Pull request merged**